src/containers/EventContainer/EventContainer.js

```diff
@@ -1 +1,13 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Nice, thanks for adding PropTypes to these components already! That will be super helpful later.